README.md

```diff
@@ -16,19 +15,18 @@ This [data source][2] plugin for [Grafana][1] allows to address Checkmk as sourc
 
 To make use of the plugin, you need to take care the correct versions are installed. You need to match both, the Grafana and the Checkmk version:
 
-- **Grafana 9.5.15 or higher** Current and previous major version of Grafana
-- **Checkmk Cloud or Checkmk MSP 2.2.0 or higher** for the signed plugin available from [Grafana][6]
-- **Checkmk 2.2.0 or higher** for the unsigned plugin available from [Github][8]
+- **Grafana 10.4.18 or higher** Current and previous major version of Grafana
+- **Checkmk 2.2.0 or higher** for the plugin available from [Github][8] or from [Grafana marketplace][6]
 
 ## Installing the plug-in
 
-The signed plug-in can be installed from the [Grafana marketplace][6] or by using the `grafana-cli`:
+The plug-in can be installed from the [Grafana marketplace][6] or by using the `grafana-cli`:
 
 ```bash
 grafana-cli plugins install checkmk-cloud-datasource
 ```
 
-For the installation of the unsigned plugin, please take a look at the official [Checkmk User Guide][3].
+To install it from the Github repository, please take a look at the official [Checkmk User Guide][3].
```

Comment:
i  think the need to install this plugin from the github repository vanished by making the signed version available for everyone, right?

The next paragraph already refers to the official docs, so i think we can remove this sentence completely?!